### ISSUE TITLE
5.0.1.10 Fluent source is 6.0.0.0 Tag

### ISSUE THREAD

**Issue by llinyu** [fluentribbon/Fluent.Ribbon#422]
Source file from https://github.com/fluentribbon/Fluent.Ribbon/archive/v5.0.1.zip

GlobalAssemblyInfo.cs

```
[assembly: AssemblyVersion("6.0.0.0")]
[assembly: AssemblyFileVersion("6.0.0.0")]
[assembly: AssemblyInformationalVersion("SRC")]

```
### Environment

- Fluent.Ribbon v5.0.1.10

Thank you!

**Comment by batzen** [comment]
Damn, i made a mistake there.
Any ideas on how to recreate a tag?

**Comment by batzen** [comment]
@llinyu ping

**Comment by batzen** [comment]
Released version 5.0.2 with a correct tag targeting master today.
